test/unit/model/jdk-install-test.js

```diff
@@ -303,37 +293,32 @@ describe('JDK installer', function() {
       expect(spy).calledWith('msiexec', installer.createMsiExecParameters());
     });
 
-    it('should catch errors during the installation', function(done) {
+    it('should catch errors during the installation', function() {
       sandbox.stub(require('child_process'), 'execFile').yields(new Error('critical error'));
 
       try {
         installer.install(fakeProgress, success, failure);
-        done();
       } catch (error) {
         expect.fail('it did not catch the error');
       }
     });
 
-    it('should call success callback if install was sucessful but redirected to different location', function(done) {
```

Comment:
this done should stay, because otherwise test just exits before installation finished.